compiler/crates/relay-bin/src/main.rs

```diff
@@ -57,6 +74,25 @@ struct Opt {
     validate: bool,
 }
 
+#[derive(Parser)]
+#[clap(about = "Run the LSP server", rename_all = "camel_case")]
```

Comment:
```suggestion
#[clap(about = "Run the language server. Used by IDEs.", rename_all = "camel_case")]
```

Reply:
Might be useful to let people know they can ignore this option for their own manual use.